Jellyfin.Server/CoreAppHost.cs

```diff
@@ -74,6 +74,7 @@ protected override void RegisterServices(IServiceCollection serviceCollection)
 
             serviceCollection.AddSingleton<IActivityManager, ActivityManager>();
             serviceCollection.AddSingleton<IUserManager, UserManager>();
+            serviceCollection.AddSingleton<IDisplayPreferencesManager, DisplayPreferencesManager>();
```

Comment:
Is it necessary to make it a singleton? No singletons depend on `IDisplayPreferencesManager` as far as I can see, but I could have missed something.

Reply:
I believe all of the ServiceStack code runs as singletons

Reply:
I see. Crappy design.